etc/config/c++.amazon.properties

```diff
@@ -861,14 +861,32 @@ compiler.zapcc190308.name=x86-64 Zapcc 190308
 
 ###############################
 # Cross GCC
-group.cross.compilers=&ppcs:&mipss:&nanomips:&mrisc32:&msp:&gccarm:&avr:&rvgcc:&xtensaesp32:&xtensaesp32s2:&xtensaesp32s3:&platspec:&kalray:&s390x:&sh:&loongarch64:&c6x:&sparc:&sparc64:&sparcleon
+group.cross.compilers=&ppcs:&mipss:&nanomips:&mrisc32:&msp:&gccarm:&avr:&rvgcc:&xtensaesp32:&xtensaesp32s2:&xtensaesp32s3:&platspec:&kalray:&s390x:&sh:&loongarch64:&c6x:&sparc:&sparc64:&sparcleon:&bpf
 group.cross.supportsBinary=true
 group.cross.groupName=Cross GCC
 group.cross.supportsExecute=false
 group.cross.licenseLink=https://gcc.gnu.org/onlinedocs/gcc/Copying.html
 group.cross.licenseName=GNU General Public License
 group.cross.licensePreamble=Copyright (c) 2007 Free Software Foundation, Inc. <a href="https://fsf.org/" target="_blank">https://fsf.org/</a>
 
+###############################
+# Cross for BPF
+group.bpf.compilers=&gccbpf
+
+# GCC for BPF
+group.gccbpf.compilers=bpfg1220
+group.gccbpf.supportsBinary=true
+group.gccbpf.supportsExecute=false
+group.gccbpf.baseName=BPF gcc
+group.gccbpf.groupName=BPF GCC
+group.gccbpf.isSemVer=true
+
+compiler.bpfg1220.exe=/opt/compiler-explorer/bpf/gcc-12.2.0/bpf-unknown-none/bin/bpf-unknown-none-g++
+compiler.bpfg1220.semver=12.2.0
+compiler.bpfg1220.objdumper=/opt/compiler-explorer/bpf/gcc-12.2.0/bpf-unknown-none/bin/bpf-unknown-objdump
+compiler.bpfg1220.demangler=/opt/compiler-explorer/bpf/gcc-12.2.0/bpf-unknown-none/bin/bpf-unknown-none-c++filt
+compiler.bpfg1220.name=bpf 12.2.0
```

Comment:
No need to be explicit here for compilers that both define `baseName` and set `isSemver: true`, tne `.name` will equal `<baseName> <semVer>`

Reply:
ahhhh sorry, I forgot to update my script that generates these lines

Reply:
https://github.com/compiler-explorer/gcc-cross-builder/pull/46 <= that should be the last time you need to remind me to remove this line /o\

Reply:
:)))